.github/workflows/main.yml

```diff
@@ -54,9 +54,9 @@ jobs:
             container_image: alpine:3.21
             host_dmd: ldmd2
           # macOS
-          - job_name: macOS 13 x64, DMD (latest)
-            os: macos-13
-            host_dmd: dmd
+          - job_name: macOS 14 arm64, LDC
+            os: macos-14
```

Comment:
`macos-14` onwards are native M1 runners. These macOS arm64 runners offer a builtin x86_64 emulator - as opposed to the *Linux* AArch64 GitHub Actions runners. This means we can still generate x86_64 code and *run* it directly on these runners.